setup.py

```diff
@@ -24,7 +24,6 @@
       package_data={"dateutil.zoneinfo": ["dateutil-zoneinfo.tar.gz"]},
       zip_safe=True,
       requires=["six"],
-      tests_require=["freezegun", "pytest"],
```

Comment:
Why did you remove this list? I could still be read from the file.

Reply:
I believe this is only ever useful for `python setup.py test`, and now that we're using `pytest` (and haven't configured `setup.py test` to run the tests yet), it's useless to have it around. There's probably some utility in setting up `setup.py test` so it works properly, but my understanding is that the `test` target is considered something of a failed experiment.

Reply:
> and now that we're using pytest (and haven't configured `setup.py` test to run the tests yet)

Ok, I can look up how other projects call pytest. I think it's quite simple.